env/tacc/group_vars/galaxydbservers.yml

```diff
@@ -68,5 +68,70 @@ telegraf_plugins_extra:
   #    - '[inputs.exec.tags]'
   #    - '  influxdb_database = "{{ galaxy_instance_codename }}_sql"'
 
+  # Some custom galaxy monitoring stuff
+  galaxy_uploaded:
+    plugin: "exec"
+    config:
+      - commands = ["/bin/bash -c 'PGDATABASE=galaxy_{{ galaxy_instance_codename }} /usr/local/bin/gxadmin iquery upload-gb-in-past-hour"]
+      - timeout = "120s"
+      - data_format = "influx"
+      - interval = "1h"
+  galaxy_lastlog:
+    plugin: "exec"
+    config:
+      - commands = ["/bin/bash -c 'PGDATABASE=galaxy_{{ galaxy_instance_codename }} /usr/local/bin/gxadmin uwsgi lastlog"]
+      - timeout = "15s"
+      - data_format = "influx"
+      - interval = "15s"
+  galaxy_jobs_queued:
+    plugin: "exec"
+    config:
+      - commands = ["/bin/bash -c 'PGDATABASE=galaxy_{{ galaxy_instance_codename }} /usr/local/bin/gxadmin iquery jobs-queued"]
```

Comment:
This is already captured by the pre-gxadmin [slurp script](https://github.com/galaxyproject/infrastructure-playbook/blob/main/env/tacc/templates/stats/galaxy_db_slurp.sh.j2), but we should remove it from there to standardize on the gxadmin method. However the old query unfortunately uses an underscore in the measurement name and the field names are `internal` and `external` instead of `unprocessed` and `processed`.

Reply:
```
ndc@galaxy-db-02% galaxy_db_slurp.sh
jobs_queued internal=1,external=117
ndc@galaxy-db-02% PGDATABASE=galaxy_main /usr/local/bin/gxadmin iquery jobs-queued
jobs-queued,group=processed count=122
jobs-queued,group=unprocessed count=1
```

Reply:
Maybe let's just move the panel for this one and `jobs_queued_internal_by_handler` to a "legacy" dashboard in Grafana?

Reply:
@nate, so should I remove galaxy_jobs_queued and jobs_queued_internal_by_handler from this file? My understanding is that we can get the data (be it with different names) from the slurp script, correct?

Reply:
No, I'd like to fully replace the slurp script with gxadmin.

Reply:
@natefoo pushed changes with timeout updated and 2 uwsgi commands removed. Let me know if there is anything else to change.